content/_policy/rules-of-use._en.md

```diff
@@ -32,7 +32,7 @@ You will only be asked to verify your identity the first time you sign into cert
 
 The Login.gov service employs recognized security and privacy best practices.
 
-Prior to using the Login.gov service, you are required to agree to these terms of service. When the Login.gov service changes its terms of service, you will be given the option to agree or to decline the updated terms of service. Similarly, when conducting certain activities in the Login.gov service, such as providing personal information for the first time, the Login.gov service may require you to re-confirm your understanding of how it uses your information.
+Prior to using the Login.gov service, you are required to agree to these terms of service. In the event Login.gov service changes its terms of service, you will be given the option to agree or to decline the updated terms of service the next time you login. Similarly, when conducting certain activities in the Login.gov service, such as providing personal information for the first time, the Login.gov service may require you to re-confirm your understanding of how it uses your information.
```

Comment:
I would have expected "log in" as two separate words here, based on my understanding of the difference of ["login" as a noun and "log in" as a verb"](https://grammarist.com/spelling/log-in-login/).

```suggestion
Prior to using the Login.gov service, you are required to agree to these terms of service. In the event Login.gov service changes its terms of service, you will be given the option to agree or to decline the updated terms of service the next time you log in. Similarly, when conducting certain activities in the Login.gov service, such as providing personal information for the first time, the Login.gov service may require you to re-confirm your understanding of how it uses your information.
```